README.rst

```diff
@@ -382,6 +382,25 @@ If your file object doesn't have one, set the ``.name`` attribute to an appropri
 Furthermore, that value has to end with a **known** file extension (see the ``register_compressor`` function).
 Otherwise, the transparent decompression will not occur.
 
+Drop-in replacement of ``pathlib.Path.open``
+--------------------------------------------
+
+Now you can natively use ``smart_open.open`` with your ``Path`` objects.
```

Comment:
OK with first part, but definitely agains of context-manager usage in this case, because:

*your variant*
```python
with patch_pathlib():
    with Path("smart_open/tests/test_data/crime-and-punishment.txt.gz").open("r") as infile:
        print(infile.readline()[:41])
```

*as I'll do that*
```python
with smart_open("smart_open/tests/test_data/crime-and-punishment.txt.gz") as infile:
    print(infile.readline()[:41])
```

shorter and simpler

Reply:
That's true.